diagnostic.md

```diff
@@ -5,74 +5,79 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+To store data and to serve up data.
```

Comment:
Good, also it allows multiple users on different devices to interact while on the same app.